contrib/vagrant/start.sh

```diff
@@ -288,17 +319,17 @@ function write_cilium_cfg() {
     if [ "$LB" = 1 ]; then
         # The LB interface needs to be the "exposed" to the host
         # interface only for master node.
-        if [ $((node_index)) -eq 1 ]; then
+#        if [ $((node_index)) -eq 1 ]; then
+#            ubuntu_1404_interface="-d eth2"
+#            ubuntu_1604_interface="-d enp0s9"
+#            ubuntu_1404_cilium_lb="--lb eth2"
+#            ubuntu_1604_cilium_lb="--lb enp0s9"
+#        else
```

Comment:
Delete these lines if they are not used, or if they should be uncommented in specific cases, add a comment specifying in which circumstances they should be uncommented.

Reply:
Fixed